web/src/SourcegraphWebApp.scss

```
@@ -93,6 +93,9 @@ $table-border-color-light: #e4e9f1;
 // Progress
 $progress-height: auto;
 
+// Collapsible
+$collapsible-expand-btn-width: 20px;
```

If I am not mistaken.
```suggestion
$collapsible-expand-btn-width: 1.25rem;
```

---

I'll try

---

works!